concept/thing/attribute.feature

```diff
@@ -11,7 +11,7 @@ Feature: Concept Attribute
     Given connection has been opened
     Given connection does not have any database
     Given connection create database: typedb
-    Given connection opens schema transaction for database: typedb
+    Given connection open schema transaction for database: typedb
```

Comment:
When we open a connection, it's `connection opens` as `it opens`. 
When we do something within a connection, it's `connection open` as `using connection, open ...` (transaction is opened not by connection but inside connection).